src/main/java/in/org/projecteka/hdaf/link/link/model/ErrorCode.java

```diff
@@ -11,7 +11,7 @@
     CareContextNotFound(1002),
     OtpInValid(1003),
     OtpExpired(1004),
-    NotHIPFound(1005);
+    HIPNotFound(1005);
```

Comment:
Same applies for this, it could be service unavailable/down/something on similar lines I believe. What do you think?